frameworks/kafka/tests/test_check_topic_list_on_zk_restart.py

```diff
@@ -0,0 +1,179 @@
+import logging
+import pytest
+
+import sdk_cmd
+import sdk_install
+import sdk_plan
+import sdk_security
+import sdk_utils
+
+from tests import config
+from tests import test_utils
+from tests import client
+
+
+LOG = logging.getLogger(__name__)
+
+
+@pytest.fixture(scope="module", autouse=True)
+def zookeeper_server(configure_security):
+    service_options = {
+        "service": {
+            "name": config.ZOOKEEPER_SERVICE_NAME,
+            "virtual_network_enabled": True,
+        }
+    }
+    ZOOKEEPER_TASK_COUNT = 2
```

Comment:
why 2? we said we will configure 1 node zk for the test right?

Reply:
This is the task count number. One for Server and the other for metrics. Configuring Zk with one node is not possible as min 3 nodes are necessary.